index.js

```diff
@@ -36,6 +64,10 @@ function mercuriusLogging (app, opts, next) {
       ? conditionalBody.bind(null, opts.logBody)
       : noop
 
+  options.buildLogMessage = typeof opts.logMessage === 'function'
+    ? customLogMessage.bind(null, opts.logMessage)
+    : noop
```

Comment:
let's avoid a func call + an array.isarray

```suggestion
    : undefined
```